features/ui/loader/loader.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from "./loader.module.scss";
+
+export function Loader() {
+  return (
+    <div data-cy="loader" className={styles.loadingSpinnerContainer}>
+      <svg
+        className={styles.loadingSpinner}
+        width="66"
+        height="66"
+        viewBox="0 0 66 66"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M62 33C62 36.8083 61.2499 40.5794 59.7925 44.0978C58.3351 47.6163 56.199 50.8132 53.5061 53.5061C50.8132 56.199 47.6163 58.3351 44.0978 59.7925C40.5794 61.2499 36.8083 62 33 62C29.1917 62 25.4206 61.2499 21.9022 59.7925C18.3837 58.3351 15.1868 56.199 12.4939 53.5061C9.801 50.8132 7.66488 47.6163 6.20749 44.0978C4.7501 40.5794 4 36.8083 4 33C4 29.1917 4.75011 25.4206 6.2075 21.9022C7.66489 18.3837 9.80101 15.1868 12.4939 12.4939C15.1868 9.801 18.3837 7.66487 21.9022 6.20749C25.4206 4.7501 29.1917 4 33 4C36.8083 4 40.5794 4.75011 44.0978 6.2075C47.6163 7.66489 50.8132 9.80101 53.5061 12.4939C56.199 15.1868 58.3351 18.3838 59.7925 21.9022C61.2499 25.4206 62 29.1917 62 33L62 33Z"
+          stroke="#F9F5FF"
```

Comment:
The color here and below are probably available in the global SCSS variables, right? You could set it to `currentColor` and then set `color: color.$...` via a CSS class